scripts/export_tasks.py

```diff
@@ -0,0 +1,40 @@
+import ast
+import os
+import pathlib
+import json
+
+lib_to_task_map = {}
+
+
+def extract_tasks(library_name, variable_name, value):
+    if variable_name == "ALLOWED_TASKS":
+        if isinstance(value, ast.Dict):
+            for key in value.keys:
+                lib_to_task_map.setdefault(library_name, []).append(key.value)
```

Comment:
Rather than using `setdefault`, a nice way is to use `defaultdict` when you define `lib_to_task_map. Then you can specify a default value

```python
from collections import defaultdict
dct = defaultdict(list)
```